nomenclature/config.py

```diff
@@ -132,7 +133,15 @@ class RegionMappingConfig(BaseModel):
     model_config = ConfigDict(populate_by_name=True)
 
 
+class DimensionEnum(str, Enum):
+    model = "model"
+    scenario = "scenario"
+    variable = "variable"
+    region = "region"
+
```

Comment:
https://github.com/openENTRANCE/openentrance/tree/main/definitions/subannual
```suggestion
     subannual = "subannual"

```